backend/app/config.py

```diff
@@ -0,0 +1,13 @@
+import functools
+
+import pydantic
+
+
+class Settings(pydantic.BaseSettings):
+    version: str
```

Comment:
would it make sense to default to dev?

Reply:
I'd prefer not to. This way if we forgot to define/pass in the env var, the app will fail to start.
So we won't run a different version and accidentally mark it as `dev`.